src/main/java/fr/dynamx/api/events/DynamXItemEvent.java

```diff
@@ -11,25 +12,25 @@ public class DynamXItemEvent extends Event {
     @Getter
     private final Side side;
     @Getter
-    private final Item item;
+    private final ItemStack stack;
 
-    public DynamXItemEvent(Side side, Item item) {
-        this.item = item;
+    public DynamXItemEvent(Side side, ItemStack stack) {
+        this.stack = stack;
         this.side = side;
     }
 
     @Cancelable
     public static class Render extends DynamXItemEvent {
         @Getter
-        private final Item item;
+        private final ItemStack stack;
```

Comment:
Should be deleted (already in DynamXItemEvent class)
(idk why it was there before)